django_short_urls/views.py

```diff
@@ -162,6 +162,10 @@ def new(request):  # pylint: disable=too-many-branches
                 status=HTTP_BAD_REQUEST,
                 message="%s may not contain a '/' character." % key)
 
+    statsd.increment(
+        'workforus.new',
+        tags=['prefix:' + unicode(params['prefix']), 'is_secure:' + unicode(request.is_secure())])
```

Comment:
It's a nice-to-have information for stats on our visitors also − your call though, let's merge if that is skipped